src/Documentation/doc/advanced/corporation/faq.md

```diff
@@ -0,0 +1,211 @@
+# FAQ
+
+> What is Corporation good for?
+
+Generating ridiculously massive income. With this income, you can buy whatever you want, e.g., augmentations, bribing factions for reputations.
+
+> How many investment rounds should I take?
+
+There are 4 rounds, and you must take all of them. Investment funds greatly boost your corporation development. Your corporation won't be able to reach its full potential in a reasonable time without them.
+
+> Investors take too much shares. Can I buy them back later?
+
+No.
+
+> Government takes too much shares when I use "Seed money" option. Can I buy them back later?
+
+No.
+
+> Why can I not buy back anything? It's ridiculous.
```

Comment:
```suggestion
> Why can't I buy back anything? It's ridiculous.
```

Reply:
AFAIK, these two usages are both grammatically correct, though "Why can't I" may sound more natural in speech. It's just a matter of style, so I can commit the change if you want.